src/Cms/AppTranslations.php

```diff
@@ -203,6 +208,18 @@ public function translations()
             return $this->translations;
         }
 
-        return Translations::load($this->root('i18n:translations'), $this->extensions['translations'] ?? []);
+        $translations = $this->extensions['translations'] ?? [];
+
+        // inject current language translations
+        if ($language = $this->language()) {
+            $translations = array_replace_recursive(
+                $translations,
+                [$language->code() => $language->translations()]
+            );
+        }
```

Comment:
I just injected the current language, but do you think we should inject translations from all languages always?